simplivity/resources/virtual_machines.py

```diff
@@ -167,7 +167,7 @@ def __init__(self, connection, resource_client, data):
     def __refresh(self):
         """Updates the VM data."""
         resource_uri = "{}/{}".format(URL, self.data["id"])
-        self.data = self._client.do_get(resource_uri)
+        self.data = self._client.do_get(resource_uri)['virtual_machine']
```

Comment:
This should be a separate commit, with and updated CHANGELOG.md update.

Reply:
Updated the CHANGELOG.md and also segregated the 2 commits, One for the fixing the issue and other to support the virtual machine power on